go.mod

```diff
@@ -3,62 +3,78 @@ module github.com/hashicorp/vault-plugin-auth-azure
 go 1.19
 
 require (
+	github.com/Azure/azure-sdk-for-go v67.2.0+incompatible
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.3.1
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.1
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v1.0.0
+	github.com/Azure/go-autorest/autorest v0.11.28
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.12
+	github.com/Azure/go-autorest/autorest/date v0.3.0
+	github.com/Azure/go-autorest/autorest/to v0.4.0
 	github.com/coreos/go-oidc v2.2.1+incompatible
 	github.com/hashicorp/go-cleanhttp v0.5.2
 	github.com/hashicorp/go-hclog v1.4.0
+	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/vault/api v1.8.3
 	github.com/hashicorp/vault/sdk v0.7.0
 	golang.org/x/oauth2 v0.4.0
 )
 
-require github.com/hashicorp/errwrap v1.1.0 // indirect
+require (
+	github.com/hashicorp/errwrap v1.1.0 // indirect
+	github.com/mitchellh/mapstructure v1.5.0 // indirect
+)
```

Comment:
Hmm, I think you can manually move these two into the require block below and then run `go mod tidy`. Not sure why they're split out 🤔

Reply:
Fixed! 🙏🏼